tests/utils/test_patterns.py

```diff
@@ -42,6 +54,12 @@ def test_wheel_file_re(filename: str, expected: dict[str, str | None]) -> None:
     assert groups == expected
 
 
+@pytest.mark.parametrize("filename", [".dist-info"])
+def test_wheel_file_re_not_matches(filename: str) -> None:
+    match = patterns.wheel_file_re.match(filename)
+    assert match is None
+
+
```

Comment:
This test is no longer of much use, is it? Thus, I think we can remove it.

Reply:
Yes, agree. Test has been removed.